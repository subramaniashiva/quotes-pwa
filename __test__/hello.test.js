import React from 'react';
import Hello from '../app/hello';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <Hello>Facebook</Hello>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});