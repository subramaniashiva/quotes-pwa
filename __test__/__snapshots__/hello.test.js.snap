// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h1>
    World!
  </h1>
  <h1>
    Test1
  </h1>
</div>
`;
